File: scratch/search_layout.js
import fs from 'fs';

const filePath = 'c:/Users/Henrique - PC/Desktop/Projetos Dev/crm-mdr/src/pages/ServiceOrders.tsx';
const content = fs.readFileSync(filePath, 'utf-8');
const lines = content.split('\n');

lines.forEach((line, idx) => {
  if (['lg:grid', 'lg:col', 'lg:flex', 'grid-cols', 'sidebar', 'activeTab', 'OsSidebar'].some(k => line.includes(k))) {
    console.log(`Line ${idx + 1}: ${line.trim()}`);
  }
});

File: scratch/search_layout.py
import re

file_path = 'c:/Users/Henrique - PC/Desktop/Projetos Dev/crm-mdr/src/pages/ServiceOrders.tsx'

with open(file_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

for idx, line in enumerate(lines):
    # Find any line that contains grid-cols, col-span, sidebar, or activeTab
    if any(k in line for k in ['lg:grid', 'lg:col', 'lg:flex', 'grid-cols', 'sidebar', 'activeTab', 'OsSidebar']):
        print(f"Line {idx + 1}: {line.strip()}")
